Doc/reference/expressions.rst

```diff
@@ -816,7 +816,7 @@ or list).  Slicings may be used as expressions or as targets in assignment or
    slicing: `primary` "[" `slice_list` "]"
    slice_list: `slice_item` ("," `slice_item`)* [","]
    slice_item: `expression` | `proper_slice`
-   proper_slice: [`lower_bound`] ":" [`upper_bound`] [ ":" [`stride`] ]
+   proper_slice: [`lower_bound`] ":" [`upper_bound`] [":" [`stride`]]
```

Comment:
I think a space between `]`'s increase readability. And the space after `[` was added for symmetry.